static-site/src/util/datasetsHelpers.js

```diff
@@ -0,0 +1,2 @@
+
+export const fetchAndParseJSON = async jsonUrl => fetch(jsonUrl).then((res) => res.json());
```

Comment:
As this is a helper function, brevity isn't critical and we could follow best practices here by enforcing headers:
```js
export const fetchAndParseJSON = async (jsonUrl) => {
  return fetch(jsonUrl, { headers: { Accept: 'application/json' }})
    .then((res) => res.json());
};
```